### ISSUE TITLE
Authz: build login page

### ISSUE THREAD

**Issue by arwhyte** [umsi-arwhyte/gffa#49]
TODO

1. copy `login.html` and `logout.html` templates from `heritagesites/templates/registration` to `gffa/apps/webapp/templates/registration`
2. Comment out the Google social login lines 16-20 (temporary) 
3. Register routes at `/gffa/urls.py` (project level)

Add 

```python
from django.contrib.auth.views import LoginView, LogoutView

...

path('login/', LoginView.as_view(), name='login'),
path('logout/', LogoutView.as_view(), {'next_page': settings.LOGOUT_REDIRECT_URL}, name='logout'),
```

4. `settings.py`: add

```python
# Login/logout redirects
LOGIN_REDIRECT_URL = '/'
LOGOUT_REDIRECT_URL = '/'
```

note: the value 'home' should also work for the above settings.

5. `base.html`: in the `navbar` two `<li>` entries:

```html
<li class="nav-item">
    <a class="nav-link" href="{% url 'login' %}">Log in</a>
</li>
{% if user.is_authenticated %}
    <li class="nav-item">
              {{ user.username }}
    </li>
    <li class="nav-item">
              <a class="nav-link" href="{% url 'logout' %}">Log out</a>
    </li>
{% endif %}

```

We can style the user name (e.g, move name to right-side end of the nav bar later--feel free to check bootstrap and sort it out yourself if you want)

See https://github.com/umsi-arwhyte/heritagesites for guidance.

**Comment by jadeharr** [comment]
I have added some updates to this issue in PR #60

**Comment by jadeharr** [comment]
Resolved by pr #58
